api/client/src/main/java/org/projectnessie/client/auth/oauth2/AuthorizationCodeFlow.java

```diff
@@ -246,7 +247,32 @@ private static HttpServer createServer(int port, HttpHandler handler) {
       throw new UncheckedIOException(e);
     }
     server.createContext(CONTEXT_PATH, handler);
-    InetAddress local = InetAddress.getLoopbackAddress();
+    InetAddress localhost = null;
```

Comment:
Why does an AWS SDK version bump affect our client auth code?

Reply:
All current PRs are affected by something else. I suspect some IPv4 vs 6 thing - but just a wild guess at this point.